pkgs/development/python-modules/pydrive2/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib,
+  python3,
+  fetchPypi,
+  google-api-python-client,
+  oauth2client,
+  pyopenssl,
+  pyyaml,
+  six,
```

Comment:
```suggestion
{ lib,
, python
, fetchPypi
, google-api-python-client
, oauth2client
, pyopenssl
, pyyaml
, six
```

Reply:
fixed in decb45313390f19369ceeea3d34700add2c10700